lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
+    if ENV.has_key?('PRYRC')
+      ENV['PRYRC']
+    elsif File.exist?(File.expand_path('~/.pryrc'))
```

Comment:
Sorry that I'm not familiar with other part of Pry. I thought this class will be loaded every time Pry is invocated, isn't it? If that's the case, it will return true once users create the file.